.github/workflows/wheels.yml

```diff
@@ -179,12 +192,13 @@ jobs:
         run: |
           $TST_CMD = @"
           python -m pip install hypothesis>=6.84.0 pytest>=7.3.2 pytest-xdist>=3.4.0;
+          ${{ matrix.python[1] == '3.13' && 'python -m pip install -i https://pypi.anaconda.org/scientific-python-nightly-wheels/simple numpy;' }}

```

Comment:
Could you put a TODO here too that this can be removed once numpy releases a 3.13 wheel?